modules/nextflow/src/main/groovy/nextflow/processor/PublishDir.groovy

```diff
@@ -498,6 +500,11 @@ class PublishDir {
     protected void processFileImpl( Path source, Path destination ) {
         log.trace "publishing file: $source -[$mode]-> $destination"
 
+        final options = new ArrayList<>(2)
+        final copyAttributes = session.config.navigate('workflow.output.copyAttributes') as Boolean
```

Comment:
I fear this `as Boolean` is a false friend and returns `true` whatever this string is. Better to isolate this into its own method returning `StandardCopyOption[]` and test it

Reply:
I have refactored to more closely match how boolean config options are typically loaded, for example [here](https://github.com/nextflow-io/nextflow/blob/49dca8bd7d2d432a229d857a77cead527da1644d/plugins/nf-google/src/main/nextflow/cloud/google/batch/client/BatchConfig.groovy#L81-L84)

In reality, many boolean config options across the codebase are loaded using `as Boolean`. I don't think we generally check for string values. In the future, the strict config loader and language server will prevent things like passing a string for a boolean anyway via type checking